packages/@lightningjs/ui-components/src/components/TextBox/TextBox.test.js

```diff
@@ -168,14 +168,53 @@ describe('TextBox', () => {
       expect(textBox._Text.text.textColor).toBe(getValidColor('#000000'));
     });
 
-    // TODO: Fix reference to Base
-    // it('should get TextDefaults when switching themes', async () => {
-    //   context.setTheme(Base);
-    //   textBox.base = context.theme;
-    //   textBox.content = 'Hello Text Defaults';
-    //   await textBox.__updateSpyPromise;
-    //   expect(textBox._Text.text.maxLinesSuffix).toBe('..');
-    // });
+    it('should get TextDefaults when switching themes', async () => {
+      const testTheme = {
+        name: 'Test',
+        typography: {
+          body1: {
+            fontFace: 'TestFont2'
+          }
+        }
+      };
+      const baseThemeValues = {
```

Comment:
Thanks for updating! And sorry I know I suggested this but do you think it would be better to reference the Base/deafult theme rather than create the baseTheme object ourselves? That way if the default theme is updated then we don't need to update the object in this unit test.

Something like this:

```js
   it('should get TextDefaults when switching themes', async () => {
      // storing default theme (since the theme hasn't been switched yet)
      const baseTheme = context.theme.typography.body1;

      const testTheme = {
        name: 'Test',
        typography: {
          body1: {
            fontFace: 'TestFont2'
          }
        }
      };

      const lightningDefaultValues = {
        fontSize: 40,
        fontStyle: 'normal',
        textAlign: 'left',
        verticalAlign: 'top',
        wordWrap: true,
        maxLines: 0,
        maxLinesSuffix: '..',
        wordWrapWidth: 0
      };

      context.setTheme(testTheme);
      textBox.content = 'Hello Text Defaults';
      await textBox.__updateSpyPromise;

      expect(context.theme.name).toBe('Test');
      expect(textBox._Text.text.fontFace).toBe(
        testTheme.typography.body1.fontFace
      );
      expect(textBox._Text.text.fontSize).toBe(baseTheme.fontSize);
      expect(textBox._Text.text.maxLinesSuffix).toBe(
        lightningDefaultValues.maxLinesSuffix
      );
      expect(textBox._Text.text.fontStyle).toBe(
        lightningDefaultValues.fontStyle
      );
    });
```

Reply:
On that note, can we reference the Lightning default text props without defining the object ourselves?

Reply:
Yeah, we can do that for baseTheme, but not sure if we can test without referencing object for LightningDefaultValues?

Reply:
Sure then let's change it for the baseTheme but keep the lightningDefault object

Reply:
And instead of `baseTheme` you can also do `defaultTheme` whichever makes the most sense